### PR TITLE
make db merge compatible

### PR DESCRIPTION
**Pull request by terencechain**
This PR implements the current DB schema to merge compatible. Meaning the DB scheme can get, post, and delete merge version of beacon state and beacon block

### CODE REVIEW

File: testing/util/altair.go [hunk 1]
```diff
@@ -270,6 +270,11 @@ func NewBeaconBlockAltair() *ethpb.SignedBeaconBlockAltair {
 	}
 }
 
+// NewBeaconBlockMerge creates a beacon block with minimum marshalable fields.
```
Comment (rauljordan) [inline testing/util/altair.go:273]: Can we put this in testing/util/merge?

**Review by nisdas — COMMENTED** [review]
Should we hold off on this ?
@jmozah is working and implementing a new schema, having the merge related code inside will make it difficult for him to implement the feature.

**Comment by terencechain** [comment]
> Should we hold off on this ? @jmozah is working and implementing a new schema, having the merge related code inside will make it difficult for him to implement the feature.

Altair code already exists today, this is the same as Altair. How much more difficult will it make the new schema given we already have to account for Altair? @jmozah

**Comment by terencechain** [comment]
> > Should we hold off on this ? @jmozah is working and implementing a new schema, having the merge related code inside will make it difficult for him to implement the feature.
> 
> Altair code already exists today, this is the same as Altair. How much more difficult will it make the new schema given we already have to account for Altair? @jmozah

Confirmed with Zahoor: 
"We already have "altair" hidden in the object already.
Its one more thing in the same place.
I dont think this would make this harder."

**Review by nisdas — APPROVED** [review]

**Pull request merged**